data/overview.yaml

```
@@ -22,6 +22,47 @@ Linear algebra:
   finite dimension representations:
     examples: ''
     Schur's lemma: ''
+  finite dimensional:
+    vector space: ''
+    canonical isomorphism with K^n: ''
+    rank of vectors: ''
+    rank of a linear map: ''
```

```suggestion
    rank of a linear map: ''
    rank of a set of vectors: ''
```
Any reason why these got flipped?